### PR TITLE
magento/devdocs#: Docs: Add attributes to Customer endpoint

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) provides documentation for the https://github.com/magento/graphql-ce/issues/765 issue.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html

## Links to Magento source code

- [CountryCodeEnum](https://github.com/magento/magento2/blob/2.3.2/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L137)
- [custom_attributes](https://github.com/magento/magento2/blob/2.3.2/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L118)
- [extension_attributes](https://github.com/magento/magento2/blob/2.3.2/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L119)
- [CustomerAddressAttribute](https://github.com/magento/magento2/blob/2.3.2/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L128)
- [CustomerAddressRegionInput](https://github.com/magento/magento2/blob/2.3.2/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L42)

Thank you!

CC: @keharper 

whatsnew
Updated changes in the 2.3.2 schema of the [customer endpoint](https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.htm).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
One more example of the `country_id` in the `customerAddressInput`
https://github.com/magento/magento2/blob/ddf42caaf25311f10b23b95a92746f99943e045e/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L29

File: guides/v2.3/graphql/reference/customer.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ Attribute |  Data Type | Description
 `suffix` | String | A value such as Sr., Jr., or III
 `telephone` | String | The telephone number
 `vat_id` | String | The customer's Tax/VAT number (for corporate customers)
+`custom_attributes` | [CustomerAddressAttribute](#CustomerAddressAttribute) | Address custom attributes
```
Comment (keharper) [inline guides/v2.3/graphql/reference/customer.md]: Place `custom_attributes` and `extension_attributes` in alphabetic order.

File: guides/v2.3/graphql/reference/customer.md [hunk 2]
```diff
@@ -323,7 +334,7 @@ Attribute |  Data Type | Description
 --- | --- | ---
 `city` | String | The city or town
 `company` | String | The customer's company
-`country_id` | String | The customer's country
+`country_id` | [CountryCodeEnum](#CountryCodeEnum) | The customer's country
```
Comment (keharper) [inline guides/v2.3/graphql/reference/customer.md]:
```suggestion
`country_id` | CountryCodeEnum | The customer's country
```

File: guides/v2.3/graphql/reference/customer.md [hunk 3]
```diff
@@ -350,6 +360,267 @@ Attribute |  Data Type | Description
...
+The `customerAddressRegionInput` object can contain the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`region_code` | String | The address region code
+`region` | String | The state or province name
+`region_id` | Int | Uniquely identifies the region
+
+
+### The list of countries codes {#CountryCodeEnum}
```
Comment (keharper) [inline guides/v2.3/graphql/reference/customer.md]: Delete this enumeration. Until the GraphQL reference information is autogenerated, I don't want to document all the enumerations, especially those that are very long, like countries, regions, and currencies. For many of the small enumerations, I list the options in the attribute description.

Comment (atwixfirster) [reply guides/v2.3/graphql/reference/customer.md]: **done**

File: guides/v2.3/graphql/reference/customer.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/reference/customer.md]: **done**

File: guides/v2.3/graphql/reference/customer.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/reference/customer.md]: **done**

**Comment by keharper** [comment]
I will merge this PR after I merge #4993. Both affect the `Customer` documentation.

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks @atwixfirster

**Pull request merged**